### PR TITLE
Export posts and comments from a given VK club

### PR DESCRIPTION
**Pull request by dimdasci**
Docker container with script that exports post and comments from a VK club for a given number of days from now.

README contains TODO section with further improvement ideas. Probably issues is a better place for a such thing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,46 @@
 # m02-export-vk-comments
 Exports comments of a given club posts. Runs as a docker container
+
+## How to install
+
+Clone the repository and go the directory.
+
+    git clone git@github.com:dimdasci/m02-export-vk-comments.git
```
Comment (PhySci) [inline README.md:8]: Could make it a bit pretty - use markdown \```some code \``` for code

**Review by PhySci — COMMENTED** [review]
Looks very nice! Well done! And I'm really impressed!

Two suggestions:
1) please add logging and replace all print's on logging message;
2) do you really need Pandas here? It seems a bit overcomplicating. If Pandas is used just to save data to cdv files, please try to avoid it. As a results, docker image will become much thinner (no pandas, no numpy, no scipy, etc).

File: README.md (hunk 1 above)
Comment (dimdasci) [reply README.md:8]: @PhySci thank for suggestions!

- [x] print was replaced by logging
- [x] pandas `to_csv` was replaced by csv module, totally agree that just writing csv with pandas is super inefficient 

I haven't  changed readme yet. [Markdown says](https://www.markdownguide.org/basic-syntax/#code-blocks): 

> To create code blocks, indent every line of the block by at least four spaces or one tab

Why do you think it's a good idea to use ```some code ``` for code instead?

Comment (PhySci) [reply README.md:8]: I mean to use code block in your example. 
Of course, ```some code``` should be replaced on your code ```git clone ...```

Comment (dimdasci) [reply README.md:8]: It seems like misunderstanding as we both are talking about code block formatting ) 

I use indentation for code block formatting as recommended in the markdown documentation mentioned above. I believe the \` or \`\`\` are good for inline code formatting. 

In fact I see no visual difference between using ``` or indentation. I was wondering if you could distinguish which of the following block is formatted with indentation and which one with  \`\`\`?

    git clone ...
    or some other code

and 

```
git clone ...
or some other code
```

Comment (PhySci) [reply README.md:8]: I got it.
 Indentation is also fine, of course!

Comment (dimdasci) [reply README.md:8]: cool, tnanx for review!

**Pull request merged**